src/views/Manual/Manual.jsx

```diff
@@ -158,7 +161,7 @@ const Manual = () => {
 			<h1>{pageTitle}</h1>
 			<div className="page-options-container" />
 			{/* ::: Intro Text ::: */}
```

Comment:
This code is crazy, @arcepaul and I discussed it and we really should not have so many things "rendering" above and below the spinner if they do not show during load.